test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 module PkgTests
 
 include("pkg.jl")
+include("repl.jl")
```

Comment:
Please put back.

The reason is that the REPL tests require internet and they are also noisy and in Base, the `pkg` testgroup is special in that it run last.